tests/python/gaia-ui-tests/gaiatest/apps/bzlite/app.py

```diff
@@ -20,16 +30,59 @@ def login(self, username, password):
         password_element.tap()
         password_element.send_keys(password)
         button_element = self.marionette.find_element(*self._button_login)
-        button_element.tap()   
+        button_element.tap()
+
+    def logout(self):
+        profile_element = self.marionette.find_element(*self._profile_icon)
+        profile_element.tap()
+        logout_element = self.marionette.find_element(*self._button_logout)
+        logout_element.tap()
+
+    def followBugs(self):
+        flagged_element = self.marionette.find_element(*self._flagged_icon)
+        flagged_element.tap()
+        filed_element = self.marionette.find_element(*self._filed_icon)
+        filed_element.tap()
+        assigned_element = self.marionette.find_element(*self._assigned_icon)
+        assigned_element.tap()
+
+    def search(self, number):
+        searchbar_element = self.marionette.find_element(*self._search_bar)
+        searchbar_element.tap()
+        number_to_enter = self.keyboard.send(number)
+        searchbar_element.send_keys(number_to_enter)
+
+    def cancel_search(self):
+        searchbar_element = self.marionette.find_element(*self._search_bar)
+        searchbar_element.tap()
+        cancelsearch_element = self.marionette.find_element(*self._button_cancel_search)
+        cancelsearch_element.tap()
+
+    def create_new_bug(self, title, description):
+        title_element = self.marionette.find_element(*self._title_input)
+        title_element.tap()
+        title_element.send_keys(title)
+        description_element = self.marionette.find_element(*self._description_input)
+        description_element.tap()
+        description_element.send_keys(description)
+        add_photo_element = self.marionette.find_element(*self._thumbnail_photo_locator)
+        add_photo_element.tap()
+        from gaiatest.apps.system.regions.activities import Activities
+        gallery = Activities(self.marionette).tap_gallery()
+        gallery.wait_for_thumbnails_to_load()
+        image = gallery.tap_first_gallery_item()
+        image.tap_crop_done()
+        submit_element = self.marionette.find_element(*self._button_submit)
+        submit_element.tap()
 
     @property
     def is_logged_in(self):
```

Comment:
I don't see the need for this is_logged_in function, the only thing it's calling is self.wait_for_dashboard_navigator_to_be_displayed() and that function is also only used once. Why not call self.wait_for_dashboard_navigator_to_be_displayed() directly then?